script.js

```diff
@@ -63,4 +63,99 @@ function loadFormData() {
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+];
+
+function openModal() {
+  const projectInfo = projects[0];
+
+  document.getElementById('modalProjectName').textContent = projectInfo.name;
+  document.getElementById('modalProjectDescription').textContent = projectInfo.cardDetail;
+
+  const projectImage = document.getElementById('modalProjectImage');
+  projectImage.src = projectInfo.image;
+
+  const skillsList = document.getElementById('modalProjectSkills');
+  skillsList.innerHTML = '';
+  projectInfo.skills.forEach((skill) => {
+    const li = document.createElement('li');
+    li.textContent = skill;
+    skillsList.appendChild(li);
+  });
+
+  document.getElementById('modalLiveVersionLink').href = projectInfo.live_version;
+  document.getElementById('modalSourceLink').href = projectInfo.link_source;
+
+  const modal = document.getElementById('modal');
+  modal.style.display = 'block';
+}
+
+openModal(0);
```

Comment:
- [ ] Great job implementing the modal. However, no matter which project I want to check I will always get the first project shown 🔢 Kindly fix that detail 🙏🏽